app/models/picture.rb

```diff
@@ -0,0 +1,16 @@
+class Picture < ActiveRecord::Base
+
+  belongs_to :user
+  delegate :email, to: :user
+  delegate :name, to: :user
+  delegate :avatar, to: :user
+  delegate :thoughts, to: :comment
+
+  has_many :comments
+
+  has_attached_file :image, :styles => { :large => "750x", :medium => "600x", :thumb => "100x" }, :default_url => "/images/:style/missing.png"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RegexpLiteral: Use `%r` around regular expression. (https://github.com/bbatsov/ruby-style-guide#percent-r)